src/main/io/ledstrip.c

```diff
@@ -1495,62 +1495,62 @@ void ledStripUpdate(timeUs_t currentTimeUs)
     }
 
     if (ledStripEnabled) {
-        if (applyProfile) {
-            ledProfileSequence_t ledProfileSequence = LED_PROFILE_SLOW;
+        if (isModeApplied) {
+            ledStripUpdateStep_t ledStripUpdateStep = LED_STRIP_UPDATE_STEP_SLOW;
 
             if (updateStartTimeUs == 0) {
                 updateStartTimeUs = currentTimeUs;
             }
 
-            switch (ledStripConfig()->ledstrip_profile) {
-#ifdef USE_LED_STRIP_STATUS_MODE
-                case LED_PROFILE_STATUS: {
-                    ledProfileSequence = applyStatusProfile(currentTimeUs);
+            switch (ledStripConfig()->ledstrip_mode) {
+#ifdef USE_LED_STRIP_DETAILED_MODE
+                case LED_STRIP_MODE_DETAILED: {
+                    ledStripUpdateStep = applyDetailedMode(currentTimeUs);
                     break;
                 }
 #endif
-                case LED_PROFILE_RACE:
-                case LED_PROFILE_BEACON: {
-                    ledProfileSequence = applySimpleProfile(currentTimeUs);
+                case LED_STRIP_MODE_RACE:
+                case LED_STRIP_MODE_BEACON: {
+                    ledStripUpdateStep = applySimpleMode(currentTimeUs);
                     break;
                 }
 
                 default:
                     break;
```

Comment:
```suggestion
                    ledStripUpdateStep = LED_STRIP_UPDATE_STEP_SLOW;
                    break;
```

Reply:
Thanks.

Reply:
(the idea is to set LED_STRIP_UPDATE_STEP_SLOW explicitly for default)